src/WinGetMCPServer/Program.cs

```diff
@@ -0,0 +1,35 @@
+// -----------------------------------------------------------------------------
+// <copyright file="Program.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace WinGetMCPServer
+{
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using ModelContextProtocol.Protocol;
+
+    internal class Program
+    {
+        private const string ServerName = "winget-mcp";
+
+        static void Main(string[] args)
+        {
+            var builder = Host.CreateApplicationBuilder();
+            builder.Logging.AddConsole(consoleOptions => { consoleOptions.LogToStandardErrorThreshold = LogLevel.Trace; });
+
+            builder.Services
+                .AddMcpServer(configureOptions =>
+                {
+                    // TODO: More options setup?
+                    configureOptions.ServerInfo = new Implementation() { Name = ServerName, Version = ServerConnection.Instance.Version };
+                })
+                .WithStdioServerTransport()
+                .WithTools<WingetPackageTools>();
+
+            builder.Build().Run();

```

Comment:
Does this handle shutdown events automagically by the framework?

Reply:
No, I doubt it, but I had thought of that.  Maybe my change to improve our quiesce story should include an event to make it easier for clients to act appropriately without needing to re-implement the entire "window to listen for shutdown messages".